x-pack/legacy/plugins/alerting_ui/np_ready/public/application/sections/actions_list/components/actions_list.tsx

```diff
@@ -0,0 +1,264 @@
+                { defaultMessage: 'Unable to delete actions' }
+              ),
+          type: 'icon',
+          icon: 'trash',
+          onClick: (item: Data) => deleteItems([item]),
+        },
+      ],
+    },
+  ];
+
+  return (
+    <section data-test-subj="actionsList">
+      <ContentWrapper>
+        <EuiSpacer size="m" />
+        <EuiSearchBar
+          onChange={({ queryText }: { queryText: string }) => setSearchText(queryText)}
+          filters={[
+            {
+              type: 'field_value_selection',
+              field: 'type',
+              name: i18n.translate('xpack.alertingUI.sections.actionsList.filters.typeName', {
+                defaultMessage: 'Type',
+              }),
+              multiSelect: 'or',
+              options: Object.values(actionTypesIndex || {}).map(actionType => ({
+                value: actionType.id,
+                name: actionType.name,
+              })),
+            },
+          ]}
+          toolsRight={[
+            <EuiButton
+              key="delete"
+              iconType="trash"
+              color="danger"
+              isDisabled={selectedItems.length === 0 || !canDelete}
+              onClick={deleteSelectedItems}
+              title={
+                canDelete
+                  ? undefined
+                  : i18n.translate(
+                      'xpack.alertingUI.sections.actionsList.buttons.deleteDisabledTitle',
+                      { defaultMessage: 'Unable to delete actions' }
+                    )
+              }
+            >
+              <FormattedMessage
+                id="xpack.alertingUI.sections.actionsList.buttons.deleteLabel"
+                defaultMessage="Delete"
+              />
+            </EuiButton>,
+            <AlertingActionsDropdown
+              key="create-action"
+              actionTypes={actionTypesIndex}
+            ></AlertingActionsDropdown>,
+          ]}
+        ></EuiSearchBar>
+
+        <EuiSpacer size="s" />
+
+        <EuiBasicTable
+          loading={isLoadingActions || isLoadingActionTypes || isDeletingActions}
+          items={data}
+          itemId="id"
+          columns={actionsTableColumns}
+          rowProps={() => ({
+            'data-test-subj': 'row',
+          })}
+          cellProps={() => ({
+            'data-test-subj': 'cell',
+          })}
+          data-test-subj="actionsTable"
+          pagination={{
+            pageIndex: page.index,
+            pageSize: page.size,
+            totalItemCount,
+          }}
+          onChange={({ page: changedPage }: { page: Pagination }) => {
+            setPage(changedPage);
+          }}
+          selection={{
+            onSelectionChange(updatedSelectedItemsList: Data[]) {
+              setSelectedItems(updatedSelectedItemsList);
+            },
+          }}
+        />
+      </ContentWrapper>
+    </section>
+  );
+};
+
+export const ContentWrapper = ({ children }: { children: React.ReactNode }) => {
```

Comment:
Is this used elsewhere?  If not, probably doesn't need to be its own component like this.  If it is, it should be in its own file I think.  Generally we stick to 1 component per file.

Reply:
Removed in https://github.com/elastic/kibana/pull/47843/commits/aca85c51ad563ec12e1670d78a8a46dbad07cae3.